docs-src/httproute.md

```diff
@@ -0,0 +1,303 @@
+extension point to express additional processing that may be performed in Gateway implementations. Some examples include
+request or response modification, implementing authentication strategies, rate-limiting, and traffic shaping.
+
+The following example adds header "my-header: foo" to HTTP requests with Host header "my.filter.com".
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-filter-1
+spec:
+  hostnames:
+    - my.filter.com
+  rules:
+    ...
+      filters:
+      - type: RequestHeaderModifier
+        requestHeaderModifier:
+          add:
+            my-header: foo
+```
+API conformance is defined based on the filter type. The effects of ordering multiple behaviors is currently
+unspecified. This may change in the future based on feedback during the alpha stage.
+
+Conformance levels are defined by the filter type:
+
+ - All "core" filters MUST be supported by implementations.
+ - Implementers are encouraged to support "extended" filters.
+ - "Custom" filters have no API guarantees across implementations.
+
+Specifying a core filter multiple times has unspecified or custom conformance.
+	
+#### ForwardTo (optional)
+
+ForwardTo defines API objects where matching requests should be sent. If unspecified, the rule performs no
+forwarding. If unspecified and no filters are specified that would result in a response being sent, a 503 error code is
+returned.
+
+The following example forwards all HTTP requests with Host header "my.filter.com" to service "my-filter-svc1":
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-filter-1
+spec:
+  hostnames:
+    - my.filter.com
+  rules:
+    - matches:
+      - path:
+          type: Prefix
+          value: /
+      forwardTo:
+      - serviceName: my-filter-svc1
+        port: 80
+```
+
+The following example forwards all HTTP requests with Host header "my.filter.com" to custom resource
+"my-custom-resource":
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-filter-1
+spec:
+  hostnames:
+    - my.filter.com
+  rules:
+    - matches:
+      - path:
+          type: Prefix
+          value: /
+      forwardTo:
+      - port: 8080
+        backendRef:
+          name: my-custom-resource
+          group: acme.io
+          kind: CustomBackend
+```
+
+The following example uses the `weight` field to forward HTTP requests with Host header "my.filter.com" equally across
+service "my-filter-svc1" and service "my-filter-svc2", i.e. traffic splitting:
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-filter-1
+spec:
+  hostnames:
+    - my.filter.com
+  rules:
+    - matches:
+      - path:
+          type: Prefix
+          value: /
+      forwardTo:
+      - serviceName: my-filter-svc1
+        port: 80
+        weight: 50
+      - serviceName: my-filter-svc2
+        port: 80
+        weight: 50
+```
```

Comment:
Also same comment as above. One additional note - it may be preferable to use weights that don't some to 100 to avoid weight being interpreted as a percentage.